### PR TITLE
feat: add method to get the pixels in a certain radar echo dot

### PR DESCRIPTION
**Pull request by seixasxbr**
# What is this PR for
Add method to return the pixels in a certain radar echo dot (angle_idx, sweep_idx)
[sc-46207]

# Checklist
- [x] Assign yourself  in the PR
- [x] Write unit tests (when relevant)

### CODE REVIEW

File: src/EchoToImageLUT.cpp [hunk 1]
```diff
@@ -129,4 +129,17 @@ void EchoToImageLUT::drawImageFromEchoes(std::vector<uint8_t> const& world_echoe
     for (long i = 0; i < static_cast<long>(world_echoes.size()); i++) {
         updateImage(radar_frame, i / m_sweep_size, i % m_sweep_size, world_echoes.at(i));
     }
-}
\ No newline at end of file
+}
+
+std::vector<cv::Point> EchoToImageLUT::getPixelsFor(int angle_unit,
```
Comment (wvmcastro) [inline src/EchoToImageLUT.cpp]: 1. keep names uniform

```suggestion
std::vector<cv::Point2i> EchoToImageLUT::getPixelsFor(int angle_idx, int sweep_idx) const
```

2. Another idea would be to return a pair of iterators for the points range something like

```cpp
...
return std::make_pair(m_data.cbegin() + point_idx, m_data.cbegin() + net_point_idx);
```
and then use:
```cpp
for(auto [pixel, last_pixel] = getPixelsFor(theta_i, sweep_j); pixel < last_pixel; pixel++) {
  // do some stuff with *pixel
}
```

**Review by wvmcastro — COMMENTED** [review]
- I din't get the `angle_unit` name when I compare it with `sweep_idx`. 
- The iterators pair return is a "weak" suggestion, don't fell obliged too

File: src/EchoToImageLUT.cpp (hunk 1 above)
Comment (seixasxbr) [reply src/EchoToImageLUT.cpp]: good call

File: src/EchoToImageLUT.hpp [hunk 2]
```diff
@@ -33,7 +33,19 @@ namespace radar_base {
...
+         * echoes to image look-up table
+         *
+         * @param angle_idx The angle index
+         * @param sweep_idx The sweep index
+         * @return std::pair<std::vector<cv::Point>::const_iterator,
+         * std::vector<cv::Point>::const_iterator>
+         */
+        std::pair<std::vector<cv::Point>::const_iterator,
+            std::vector<cv::Point>::const_iterator>
+        getPixels(int angle_idx, int sweep_idx);
```
Comment (wvmcastro) [inline src/EchoToImageLUT.hpp]: When I see a _getter_ method, I expect it to not change its object. I.e. that it is constant
```suggestion
        getPixels(int angle_idx, int sweep_idx) const;
```

**Review by kapeps — APPROVED** [review]

**Review by higor-melo — APPROVED** [review]

**Review by wvmcastro — APPROVED** [review]

**Pull request merged**